### PR TITLE
Attempt to fix share instagram on iOS

### PR DESCRIPTION
**Pull request by henrytao-me**
This PR attempts to fix share Instagram on iOS based on suggestion here https://stackoverflow.com/questions/11393071/how-to-share-an-image-on-instagram-in-ios

Note that: I don't have iOS device with Instagram in order to verify this.

Resolves https://github.com/CovidShield/mobile/issues/62

### Demo

https://drive.google.com/file/d/1BdK6lcchWb7lhRXd67LGYwgyOzNJQM4K/view?usp=sharing

### CODE REVIEW

**Comment by jordanAtShopify** [comment]
I also can't verify though

**Comment by kpeatt** [comment]
This doesn't work for me on device. I don't get the option to select instagram on the page and I can't select it in the share sheet. I can see Instagram on my other sharing apps.

**Comment by henrytao-me** [comment]
Fixed. Posted a video here https://drive.google.com/file/d/1v37L3s8_l-Whh7zAOPbCKSepjsX4WPuP/view?usp=sharing

For instagram, it only allows share image. What image should we share? I am putting CovidShield logo as an example right now. 

cc @kpeatt @AdamWhitcroft

**Comment by kpeatt** [comment]
It works!

File: src/locale/translations/en.json [hunk 1]
```diff
@@ -101,6 +101,7 @@
     "Platform-instagram": "Share to Instagram",
     "More": "More apps",
     "Message": "Join me in helping to slow the spread of COVID-19. Download the COVID Shield app: https://covidshield.app",
+    "InstagramImageUrl": "https://user-images.githubusercontent.com/5274722/84658989-b9ba6b00-aee4-11ea-84d4-d840527467b4.png",
```
Comment (henrytao-me) [inline src/locale/translations/en.json:104]: Sharing on Instagram iOS requires url, so I use github as the image host for now.

File: src/screens/sharing/Sharing.tsx [hunk 2]
```diff
@@ -34,7 +34,13 @@ export const SharingScreen = () => {
         [platform]: () => {
           switch (platform) {
             case 'instagram': {
-              shareInstagramStory(theme.colors.mainBackground);
+              shareInstagramStory(
+                theme.colors.mainBackground,
+                Platform.select({
+                  android: i18n.translate('Sharing.Message'),
```
Comment (henrytao-me) [inline src/screens/sharing/Sharing.tsx:40]: Due to limitation of Instagram API on Android, it cannot share as a story (which is url). So, we use message instead.

**Pull request merged**